README.md

```diff
@@ -2851,6 +2845,53 @@ country:
   employees. José Ancer provides a thoughtful
   [overview](http://siliconhillslawyer.com/2016/06/23/founder-compensation-cash-equity-liquidity/).
 
+🚧 This section currently mostly covers startups;
+what later-stage resources are available?
+
+### Typical Startup Advisor Equity Levels
+
+🄳 **Advisors** are people who have extensive or unique experience that help a company in a
+formal or informal capacity.
+They are often brought on by startups that want a recognized name, experience, access to a
+network, or skills not already on a small team.
+Sometimes advisors act as mentors to founders.
```

Comment:
"Startups bring in advisors when they want a recognized name, specific experience or skills, or access to a network."